src/Data/Generator.purs

```diff
@@ -313,15 +315,17 @@ toReturnType constant outputs' = do
       import' "Network.Ethereum.Web3.Types.Types" [IType "HexString"]
       pure "Web3 p e HexString"
     else do
+      import' "Network.Ethereum.Web3.Types" [IType "CallError"]
+      import' "Data.Either" [IType "Either"]
       outputs <- for outputs' toPSType
       out <- case length outputs of
         0 -> pure "Unit"
         1 -> pure $ unsafePartial $ unsafeIndex outputs 0
         _ -> do
           let tupleType = "Tuple" <> show (length outputs)
           import' "Network.Ethereum.Web3.Solidity" [IType tupleType]
-          pure $ "(" <> tupleType <> " " <> joinWith " " outputs <> ")"
-      pure $ "Web3 p e " <> out
+          pure $ tupleType <> " " <> joinWith " " outputs
+      pure $ "Web3 p e " <> "(Either CallError (" <> out <> "))"
```

Comment:
What are the reasons for this change?
They way pares where before was correct:
- for Unit you don't need parens
- when you have one element in outputs you also don't need parens as `toPSType` will add them if needed

Reply:
restored it here https://github.com/f-o-a-m/purescript-web3-generator/pull/17